### PR TITLE
enhance: Support L0 import

### PR DESCRIPTION
**Pull request by bigsheeper**
issue: https://github.com/milvus-io/milvus/issues/33157

### CODE REVIEW

**Comment by bigsheeper** [comment]
/hold
need to remove debug log

**Comment by bigsheeper** [comment]
rerun ut

File: internal/core/unittest/CMakeLists.txt [hunk 1]
```diff
@@ -88,7 +88,7 @@ endif()
 if (DEFINED AZURE_BUILD_DIR)
     set(MILVUS_TEST_FILES
                 ${MILVUS_TEST_FILES}
-                test_azure_chunk_manager.cpp
+#                test_azure_chunk_manager.cpp
```
Comment (czs007) [inline internal/core/unittest/CMakeLists.txt:91]: revert this

File: internal/datanode/services.go [hunk 2]
```diff
@@ -467,17 +478,19 @@ func (node *DataNode) QueryPreImport(ctx context.Context, req *datapb.QueryPreIm
 	}
 	status := merr.Success()
 	task := node.importTaskMgr.Get(req.GetTaskID())
-	if task == nil || task.GetType() != importv2.PreImportTaskType {
-		status = merr.Status(importv2.WrapNoTaskError(req.GetTaskID(), importv2.PreImportTaskType))
+	if task == nil {
+		status = merr.Status(importv2.WrapNoTaskError(req.GetTaskID()))
```
Comment (czs007) [inline internal/datanode/services.go:482]: rename to WrapTaskNotFoundError

File: internal/util/importutilv2/option.go [hunk 3]
```diff
@@ -76,3 +77,11 @@ func IsBackup(options Options) bool {
 	}
 	return true
 }
+
+func IsL0Import(options Options) bool {
+	isBackup, err := funcutil.GetAttrByKeyFromRepeatedKV(L0Import, options)
```
Comment (czs007) [inline internal/util/importutilv2/option.go:82]: isBackup rename to isL0

**Comment by bigsheeper** [comment]
rerun ut

**Comment by zhuwenxing** [comment]
@bigsheeper 

This pull request should also be merged into the 2.3 branch.

In the 2.3 branch, after restoration, when a collection has duplicate primary keys, the latest inserted data will not be queried.
https://github.com/zhuwenxing/milvus-backup/actions/runs/10193209545/job/28197561271

**Pull request merged**